controllers/sensor/resource.go

```diff
@@ -212,12 +227,24 @@ func buildDeployment(args *AdaptorArgs, eventBus *eventbusv1alpha1.EventBus) (*a
 				},
 			},
 		})
+
 		volumeMounts = append(volumeMounts, corev1.VolumeMount{
 			Name:      "auth-volume",
 			MountPath: common.EventBusAuthFileMountPath,
 		})
 	}
 
+	if args.Sensor.Spec.LiveReload {
+		volumes = append(volumes, corev1.Volume{
+			Name: "sensor-config-volume",
+			VolumeSource: corev1.VolumeSource{
+				ConfigMap: &corev1.ConfigMapVolumeSource{LocalObjectReference: corev1.LocalObjectReference{
+					Name: fmt.Sprintf("sensor-cm-%s", args.Sensor.Name),
+				}}},
+		})
+		volumeMounts = append(volumeMounts, corev1.VolumeMount{Name: "sensor-config-volume", MountPath: "/sensor-definition"})
```

Comment:
What is set mountpath to `/etc/sensor` to match this [eventbus mount path](https://github.com/intuit-data-os/argo-events/blob/master/common/common.go#L47)? Also, if we use a constant in `common.go` we could use that constant to read from the same file in the sensor.